src/nvim/testdir/test_usercommands.vim

```diff
@@ -0,0 +1,44 @@
+" Tests for user defined commands
+
+" Test for <mods> in user defined commands
+function Test_cmdmods()
+  let mods = ''
+
+  command! -nargs=* MyCmd let mods .= '<mods> '
```

Comment:
a command cannot modify a local variable. I actually am not sure how this test would pass.